README.md

```diff
@@ -32,6 +32,24 @@ manage a new cloud-native application. This enables Crossplane users to:
 The full vision and architecture of the Crossplane project is described in our
 [architecture document].
 
+## Infrastructure Stacks
```

Comment:
Should this be limited to infrastructure stacks?

Reply:
I am not sure there are any app stacks to point to at the time, but we could add a section when more are available.